### ISSUE TITLE
getChapter call fails on some chapters

### ISSUE THREAD

**Issue by PeterCharalambous** [ejnshtein/mangadex-api#27]
The `client.chapter.getChapter` call fails with the below on some chapters/manga.

Using ID **1161820** works however using ID **1158194** results in the error below

```
TypeError: chapter.pages.map is not a function
    at Object.normalizeChapter (\node_modules\mangadex-api\dist\lib\normalize.js:17:26)
    at ChapterResolver.getChapter (\node_modules\mangadex-api\dist\api\chapter.js:15:28)
    at async Object.getChapter (\controllers\manga.js:69:19)
```

**Comment by ejnshtein** [comment]
True, some chapters don't have pages in them and code fails.
Will fix asap!

**Comment by ejnshtein** [comment]
Should be fixed in v4.5.1!
